source/DistanceMap.cpp

```diff
@@ -154,7 +154,7 @@ void DistanceMap::Init(const System *center, const Ship *ship)
 			hyperspaceFuel = 0.;
 		
 		// If this ship has no mode of hyperspace travel, bail out.
```

Comment:
Good point. I almost wonder if the better fix is to (instead of `!player`) verify that there are no wormholes in the relevant ship's system, as wormholes (while not explicitly 'hyperspace' travel, are an intersystem transportation mechanism.